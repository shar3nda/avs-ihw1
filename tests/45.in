10
1 2 8 0 9 2 0 2 3 3