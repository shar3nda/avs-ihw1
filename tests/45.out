0 9 2 0 2 3 3